src/request.js

```diff
@@ -63,6 +63,9 @@ export default class Request {
 
 		if (init.body != null) {
 			const contentType = extractContentType(this);
+			if (contentType === 'application/x-www-form-urlencoded') {
+				this.body = String(this.body);
+			}
```

Comment:
This belongs to `consumeBody()` and `writeToStream()` in `src/body.js`, where the appropriate actions for other types of `body`types are listed as well. You might also have to implement `getTotalBytes()` there.